.nvmrc

```diff
@@ -0,0 +1 @@
+v18.16.1
```

Comment:
Added this recommendation so it's clear which specific node version is being used/targeted in tests, and expected to be used in development.